setup.py

```diff
@@ -17,14 +17,23 @@
 
 import os
 
+from xml.etree import ElementTree
+
 from setuptools import find_packages
 from setuptools import setup
 
 package_name = 'cross_compile'
 
+
+def read_version_from_package_xml():
+    tree = ElementTree.parse('package.xml')
+    version = tree.find('version')
```

Comment:
`tree.find` returns the value or `None`. 
Should we handle if its `None`?

Reply:
`package.xml` schema forces a version tag to exist. It's *good* to fail if the `package.xml` is incorrect, and we should not handle this case.